packages/console/src/pages/OrganizationDetails/index.tsx

```diff
@@ -70,14 +67,19 @@ function OrganizationDetails() {
     (!jitEmailDomains.data && !jitEmailDomains.error) ||
     (!jitRoles.data && !jitRoles.error) ||
     (!jitSsoConnectors.data && !jitSsoConnectors.error);
+
   const error =
     organization.error ?? jitEmailDomains.error ?? jitRoles.error ?? jitSsoConnectors.error;
 
   return (
-    <DetailsPage backLink={pathname} backLinkTitle="organizations.title" className={styles.page}>
+    <DetailsPage
+      backLink={pathname}
+      backLinkTitle="organizations.title"
+      className={styles.page}
+      isLoading={isLoading}
+      error={error}
+    >
       <PageMeta titleKey="organization_details.page_title" />
-      {isLoading && <Skeleton />}
```

Comment:
Should we remove the skeleton component, then? I can't quite recall if the skeleton is anything different than the one used in `<DetailsPage />` component. Can you please also take a look?

Reply:
They're the same compoent.
<img width="509" alt="image" src="https://github.com/user-attachments/assets/e2c80e0c-1714-4261-8458-afd9b1e34e78">